src/Uno.UWP/Devices/Bluetooth/BluetoothDevice.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;

```

Comment:
Please use the default formatter for this file, the spacing and indentation is off.

Reply:
^K D applied (sequence found on Internet)